utils/storage.ts

```diff
@@ -41,20 +41,6 @@ export async function getMeta(
   return new Uint8Array(data);
 }
 
-export async function getVersionMetaJson(
```

Comment:
Was this never used?

Reply:
its something i forgot to remove in #304